zealot/zed/types/type-map.ts

```diff
@@ -13,15 +13,17 @@ export class TypeMap implements ContainerTypeInterface {
     return `|{` + typeId(keyType) + ":" + typeId(valType) + "}|"
   }
 
-  create(value: [Value, Value][], typedefs) {
+  create(value: [Value, Value][] | null, typedefs) {
```

Comment:
Values can always be null. This was another bug the test caught. The Map and Union types are more rare and have not been battle tested as much so it makes sense these bugs are here.